src/test/scala/com/cloudera/sparkts/parsers/YahooParserSuite.scala

```diff
@@ -14,7 +14,7 @@
  */
 package com.cloudera.sparkts.parsers
 
-import org.joda.time.DateTimeZone
+import codes.reactive.scalatime.ZoneId
```

Comment:
I think, in general, probably better to import java.time.ZoneId than codes.reactive.scalatime.ZoneId.